skiko/src/jvmMain/kotlin/org/jetbrains/skiko/PlatformOperations.kt

```diff
@@ -30,7 +33,10 @@ internal val platformOperations: PlatformOperations by lazy {
                     return component.graphicsConfiguration.defaultTransform.scaleX.toFloat()
                 }
 
-                override fun createHardwareRedrawer(layer: HardwareLayer) = MacOsRedrawer(layer)
+                override fun createHardwareRedrawer(layer: HardwareLayer) = when(renderApi) {
```

Comment:
Maybe we rename createHardwareRedrawer to createRedrawer? Because RasterRedrawer isn't hardware.

Reply:
Done.